requirements.txt

```diff
@@ -1,69 +1,44 @@
-alabaster==0.7.10
 alembic==0.9.9
 Babel==2.6.0
 bcrypt==3.1.4
 beautifulsoup4==4.6.0
 cffi>=1.7.0
```

Comment:
Ah, sorry, one more thing :) I think it would be nice if we could replace the ``>=`` conditions with specific versions -- what do you think?

Reply:
fixed in d4ae1646